src/lib/yang/schema.lua

```diff
@@ -811,7 +811,17 @@ function resolve(schema, features)
                   assert(not node.body[k], 'duplicate identifier: '..k)
                   node.body[k] = v
                end
-               -- TODO: Handle refine and augment statements.
+               for _,refine in ipairs(v.refines) do
+                  local target = node.body[refine.node_id]
+                  assert(target, 'missing refine node: '..refine.node_id)
+                  -- FIXME: Add additional "must" statements.
+                  for _,k in ipairs({'config', 'description', 'reference',
+                                     'presence', 'default', 'mandatory',
+                                     'min_elements', 'max-elements'}) do
```

Comment:
Question: Is it 'min_elements' correct? Shouldn't be 'min-elements'? Probably both are valid, although is worth noticing this is the only 'min_elements' literal in the whole file.

Reply:
Oh good catch, thanks!